src/components/films-catalog/films-catalog.tsx

```diff
@@ -0,0 +1,36 @@
+import FilmsList from '../films-list/films-list';
+import GenresList from '../genres-list/genres-list';
+import { useAppDispatch, useAppSelector } from '@/hooks';
+import { chooseGenreAction, resetGenreAction} from '@/store/action';
+import { Genres } from '@/utils/const';
+import { useState } from 'react';
+
+
+export default function FilmsCatalogue() {
+  const genre = useAppSelector((state) => state.genre);
+  const films = useAppSelector((state) => state.films);
+  const dispatch = useAppDispatch();
+
+  const [activeGenre, setActiveGenre] = useState(genre);
+
+  const handleSortClick = (value: Genres) => {
+    setActiveGenre(value);
+    if(value === Genres.AllGenres) {
+      return dispatch(resetGenreAction());
+    }
+    dispatch(chooseGenreAction({genre: value}));
```

Comment:
if или циклы обычно отделяют пустой строкой
```suggestion
    setActiveGenre(value);
    
    if(value === Genres.AllGenres) {
      return dispatch(resetGenreAction());
    }
    
    dispatch(chooseGenreAction({genre: value}));
```